populate.js

```diff
@@ -75,4 +74,6 @@ const savePhotos = async (primaryUrls, productPhotosUrls) => {
   });
 }
 
-populateDb();
\ No newline at end of file
+populateDb();
+
+module.exports = savePhotos;
```

Comment:
In below code, the code inside if and else looks similar. If is using data from host and Else is using faker data ? If not, what is the logic here? Adding comments might be helpful 

```
_if (i < 42) {
} else { }_
```
--------------------
Below code repeats in if and else both. You might want to move it outside if /else ?
```
_let item = {
        id: i + 1000,
        primaryUrl: primaryUrls[i],
        productUrls: images,
        featuresUrls: features
      }_
```

-----------
Why is DB being dropped only for <= 100? If it is more than 100, you would like to keep the data?

```
_if (checkForPreviousSeedCount <= 100) {
    await db.Photo.db.dropDatabase();
  };_
```

Reply:
That's correct, I believe I did add a comment here on my latest commit.    
 // number of available photos in host service is 300. 7 pictures per product 300/7 = 42.8.
 // the rest of the product pictures are mocked using faker.

Reply:
Also, that does look repetitive, I'll look into this

Reply:
Closing as you have made changes.